src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/product_expiration_date
+spring.datasource.username=postgres
+spring.datasource.password=ZaRAPer1404
```

Comment:
It's better to add those properties via environment variables https://education.launchcode.org/gis-devops/configurations/02-environment-variables-intellij/index.html